src/utils/parseQueryParams.js

```diff
@@ -0,0 +1,46 @@
+export function parseQueryParams() {
+  var queryString = window.location.search;
+  if (queryString === '') return {};
+  var keyValPairs = [];
+  var params = {};
+  queryString = queryString.replace("?", "");
+  queryString = queryString.replace(/%20/g, " ");
+  queryString = queryString.replace(/%2C/g, ",")
```

Comment:
Instead of using regex to clean up URI encoded strings, we should use the builtin JS function to do so: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/decodeURI

This way we won't have to update this when new edge cases pop up